### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by ViktorGubarevich**
Homework lesson-15 Task A, B, C.

### CODE REVIEW

File: script-task-b.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+alert(family);
+console.log(family);
+alert(date);
+console.log(date);
+alert(girlFriend);
+console.log(girlFriend);
+
+let showMessage = function showMessage(girlFriendFirstName, girlFriendLastName) {
+    alert('Hi. I am Viktor!');
+    console.log('Welcome, ' + girlFriendFirstName + ' ' + girlFriendLastName);
```
Comment (akhovbok) [inline script-task-b.js:28]: строковый шаблон ``

File: script-task-c.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+// Task C
+
+let years = prompt('How old are you?', '');
```
Comment (akhovbok) [inline script-task-c.js:3]: const

File: script-task-b.js (hunk 1 above)
Comment (ViktorGubarevich) [reply script-task-b.js]: Completed.

File: script-task-c.js (hunk 2 above)
Comment (ViktorGubarevich) [reply script-task-c.js]: Completed.

**Pull request merged**